build.go

```diff
@@ -29,12 +31,18 @@ type EnvironmentConfig interface {
 	GetValue(key string) string
 }
 
+//go:generate faux --interface SBOMGenerator --output fakes/sbom_generator.go
+type SBOMGenerator interface {
+	Generate(dir string) (sbom.SBOM, error)
+}
+
 func Build(projectPathParser PathParser,
 	bindingResolver BindingResolver,
 	buildManager BuildManager,
 	clock chronos.Clock,
 	environment EnvironmentConfig,
-	logger scribe.Logger) packit.BuildFunc {
+	logger scribe.Logger,
```

Comment:
@ryanmoran The logger is used to initialize the `EnvironmentConfig` and `BindingResolver` objects and it doesn't take an emitter. Do you still want me to make this change?
For ref: https://github.com/paketo-buildpacks/npm-install/blob/main/run/main.go#L22

Reply:
You'll need to use an Emitter to be able to use the newly added SBOM logging methods like `logger.GeneratingSBOM`. It is clear that this buildpack needs to be updated to use the Emitter instead of the Logger, but you can make the smallest change required here and we can think about addressing that technical debt in the future.

Reply:
👍